chef/cookbooks/swift/templates/default/container-server.conf.erb

```diff
@@ -11,7 +11,6 @@ user = <%= node[:swift][:user] %>
 #
 # Use an integer to override the number of pre-forked processes that will
 # accept connections.
-# workers = auto
 workers = 2
```

Comment:
In some other barclamps, we keep the default commented values. Not sure if we want to do this here?